docs/DevelopmentSetup/visual_studio_win_example.rst

```diff
@@ -113,12 +147,40 @@ To match this, the example point config uses an environment variable in the libr
 RothC example
 =============
 We also have a RothC example for point level simulations. Inorder to run this example, you may modify the following arguments in the above test settings command arguments.
-These arguments will point at the right configuration files for RothC.
+These arguments will point at the right configuration files for RothC. Please follow the following steps to set the correct configuration -
 
-::
+* Build the debug configuration ``ALL_BUILD`` target by right clicking the ``ALL_BUILD`` node and selecting ``Build``. Then right click the the ``moja.flint.example.rothc`` node and select ``Set as Startup Project`` then right click again and select properties. Navigate to Configuration ``Properties/Debugging`` properties pane and configure the following:
 
-    # Command Args
-    --config config/point_rothc_example.json --config config/$(Configuration)/libs.base_rothc.win.json --logging_config logging.debug_on.conf
+  * Command: ``C:\Development\moja-global\FLINT\Source\build\bin\Debug\moja.cli.exe``
+
+  * Command Arguments:
+  ::
+
+      --config config\point_example.json --config config\debug\libs.base.win.json --logging_config logging.debug_on.conf
+
+  * Working Directory: ``$(SolutionDir)..\..\Run_Env``
+
+  * Environment:
+  ::
+
+
+      PATH=C:\Development\moja-global\FLINT\Source\build\bin\Debug;%PATH% LOCAL_LIBS=C:\Development\moja-global\FLINT.Example\Source\build\bin\Debug
+
+.. figure:: ../images/installation_vs2019_flint.example/VS2019_rothcproperties.jpeg
+  :width: 600
+  :align: center
+  :alt: Alternative text
+
+  ``ALL_BUILD`` properties page for ``moja.FLINT.example.rothc``
+
+You should now be able to select ``Debug->Start Debugging`` to start a debug run of the RothC example. You should see something like the following:
+
+.. figure:: ../images/installation_vs2019_flint.example/VS2019_debugrothc.jpeg
+  :width: 600
+  :align: center
+  :alt: Alternative text
```

Comment:
What is the primary takeaway from the debug screenshot? Is there something a new reader should look for?

Reply:
@aornugent @shubhamkarande13 will connect with you to get an idea of new screenshots we shall be using :sweat_smile: